substratest/assets.py

```diff
@@ -1,5 +1,5 @@
 import abc
-import enum
+from enum import Enum, auto
```

Comment:
Why changing the import? I think using `enum.auto()` is more explicit than using `auto()` directly.

Reply:
Yes, you're right, I thought that it was cleaner this way